gem/lib/illuminator/automation-builder.rb

```diff
@@ -27,7 +27,7 @@ def buildForAutomation sdk, hardwareID
         @arch = @arch || 'i386'
       else
         @sdk = sdk || 'iphoneos'
-        @arch = @arch || 'armv7'
+        #@arch = @arch || 'armv7'
```

Comment:
Don't remove this line... I think the proper fix is in xcodebuilder.rb to
unset the arch parameter if destination is specified.

Near here:
https://github.com/paypal/Illuminator/blob/a16bd59a7be5f8e938b64ce2fbfde97f9021bfb6/gem/lib/illuminator/xcode-builder.rb#L72
On May 2, 2015 9:16 PM, "Boris Erceg" <notifications@github.com> wrote:

> In gem/lib/illuminator/automation-builder.rb
> <https://github.com/paypal/Illuminator/pull/52#discussion_r29550326>:
>
> > @@ -27,7 +27,7 @@ def buildForAutomation sdk, hardwareID
> >          @arch = @arch || 'i386'
> >        else
> >          @sdk = sdk || 'iphoneos'
> > -        @arch = @arch || 'armv7'
> > +        #@arch = @arch || 'armv7'
>
> Can you please remove line instead of just commenting? Thanks
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/paypal/Illuminator/pull/52/files#r29550326>.
>